w07-ruby-on-rails-cont/d3-homework/README.md

```diff
@@ -0,0 +1,88 @@
+# How to Collaborate with Github
+
+Github is a powerful tool for versioning and collaborating on code. In this article we will focus on how to collaborate using Github by **creating pull requests** for repos we would like to make changes for. There are two main scenarios for collaborating on coding projects: either you are a collaborator, or not. If you are a collaborator on the repo, you can clone the project to create pull requests (but be careful -- you could also push directly to origin/master, which is not recommended). If you are not a collaborator on the repo, you will have to fork the project in order to make a pull request.
```

Comment:
minor: 
for repos we would like to make changes for.
for repos we would like to make changes to.

Glad you threw in the origin/master syntax.  It's time they saw it.